lib/gui/app/modules/progress-status.ts

```diff
@@ -62,6 +62,12 @@ export function fromFlashState({
 		} else {
 			return { status: 'Finishing...' };
 		}
+	} else if (type === 'downloading') {
+		if (percentage == null) {
+			return { status: 'Downloading...' };
+		} else if (percentage < 100) {
```

Comment:
Can it be > 100 ?

Reply:
I don't think so, I remember we had an issue with the `flashing` type continuously going up but it shouldn't be the case with the `downloading` type. I'll check how it behaves with the condition removed